### DISCUSSION TITLE
How to show data inside Voyager admin dashboard depending on subdomain?

### DISCUSSION THREAD

**Discussion by AlexDevSev** [thedevdojo/voyager#5608] [Q&A]
Hi everyone!

I managed to set up Laravel Subdomain routes in our project so that when user enters Voyager Admin at subdomain1.oursite.com they are successfully logged in into Voyager Admin panel. However, when browsing, for example, Projects model in the left menu. user sees all records (including of other subdomains). I tried to experiment with the file in vendor/tcg/voyager/src/http/conrollers/VoyagerBaseController.php and tried to add Subdomain routing into it, but still can't get it to work. 

Is there any way to 'filter' model data in the left admin menu depending on which subdomain user logged in at the moment? 

Thanks for any help!

**Comment by fletch3555** [comment]
What you're looking for is multi-tenancy.  Voyager doesn't provide any specific functionality for that out of the box, but should work fine if you implement it within a multi-tenant app (meaning, one that has proper isolation for database, cache, routes, etc).  I can't recommend any specific packages to do this for you, but I do know of one person that was able to do so with [hyn/multi-tenant](https://github.com/tenancy/multi-tenant).  I imagine it would work with [tenancy/tenancy](https://github.com/tenancy/tenancy) and others just as well
